CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - IME input swallowed after triggering a key binding
 - Crash on Wayland due to non-standard fontconfig configuration
 - Search without vi mode not jumping properly between all matches
+- IME composition preview not appearing on Windows
```

Comment:
This needs to go into the 0.8.0-dev release list. I've just made a PR to bump the master to the new version (https://github.com/alacritty/alacritty/pull/4659), once that's merged you can rebase on to that and add it in the proper place.